primitives/io/src/lib.rs

```diff
@@ -1561,15 +1561,6 @@ mod tests {
 		let mut ext = BasicExternalities::default();
 		ext.register_extension(TaskExecutorExt::new(TaskExecutor::new()));
 		ext.execute_with(|| {
-			// invalid ed25519 signature
-			crypto::start_batch_verify();
-			crypto::ed25519_batch_verify(
-				&Default::default(),
-				&Vec::new(),
-				&Default::default(),
-			);
-			assert!(!crypto::finish_batch_verify());
```

Comment:
I'd expect 1/8th of the possible messages makes this succeed, so this test was mostly useless anyways.